sanity/agent/cmd.py

```diff
@@ -20,4 +21,4 @@ def syscmd(message="", timeout=300):
         print(f"command {message} timeout, timeout={timeout}")
         return FAILED
 
-    return SUCCESS
+    return SUCCESS, result
```

Comment:
```suggestion
    return SUCCESS, result.stdout
```
I think we only care the stdout. Moreover, we already check the status of run function.